docs/rules/README.md

```diff
@@ -229,6 +228,7 @@ These rules are only relevant to ES6 environments.
 * [no-const-assign](no-const-assign.md) - disallow modifying variables that are declared using `const` (recommended)
 * [no-dupe-class-members](no-dupe-class-members.md) - disallow duplicate name in class members (recommended)
 * [no-new-symbol](no-new-symbol.md) - disallow use of the `new` operator with the `Symbol` object (recommended)
+* [no-restricted-imports](no-restricted-imports.md) - restrict usage of specified ES6 imports
```

Comment:
The parallel grammar reads well.

Your question gave me a reason to improve my research skills 🤓 Only change I recommend is replace the word statements at the end of each description:
* …by `require` function [see https://nodejs.org/api/globals.html#globals_require and https://nodejs.org/api/modules.html#modules_addenda_package_manager_tips]
* …by `import` declaration [see http://www.ecma-international.org/ecma-262/6.0/index.html#sec-imports and https://github.com/eslint/eslint/blob/master/lib/rules/no-restricted-imports.js#L22]

For your info, making the description of each rule in the list consistent with the main heading in its page will probably happen in several weeks, so you do not need to worry about that now.